### PR TITLE
Docker support for all templates

### PR DESCRIPTION
**Pull request by Argus416**
Docker support for all templates + support nodemon for autoreload

### CODE REVIEW

File: packages/yonode-templates/JS-MongoDB-Mongoose-NoAuth-Template/.env [hunk 1]
```diff
@@ -8,12 +8,15 @@ SERVER_PORT=
...
+# For example, 'mongodb://root:example@localhost:27017/'.
+# If you are using docker-compose, the database URL should be 'mongodb://root:toor@db:27017/' where 'db' is the name of the database service.
 DATABASE_URL=
 
+
 # Specify the name of the database to which the application will connect.
 # If not specified, the default database 'yonodeDB' will be used.
 # Default: yonodeDB
-DATABASE_NAME=
+DATABASE_NAME=yonodeDB
```
Comment (sharafdin) [inline packages/yonode-templates/JS-MongoDB-Mongoose-NoAuth-Template/.env:19]: bro why you make the db name into yonode ? it has already in config file ].

File: packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env [hunk 2]
```diff
@@ -8,15 +8,10 @@ SERVER_PORT=
...
-# Format: protocol://username:password@host:port/databaseName
+# For example, 'mysql://username:password@localhost:5432/database'.
+# If you are using docker-compose, the database URL should be 'mysql://username:password@mysql-db:5432/database' where 'db' is the name of the database service.
 DATABASE_URL=
 
-# Authentication and Security Configuration
-# -----------------------------------------
-# Define a secret key for JWT (JSON Web Token) authentication. This key is used to sign and verify JWTs for secure data transfer and access control.
-# The JWT_SECRET_KEY should be a long, random string that is kept secure to ensure token integrity and prevent unauthorized access.
-JWT_SECRET_KEY=
```
Comment (sharafdin) [inline packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env:18]: why you removed these

**Review by sharafdin — COMMENTED** [review]
you did some changes to .env file that is not clear. please clarify for me

File: packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env (hunk 2 above)
Comment (Argus416) [reply packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env:18]: it is no-auth template so it doesn't matter if we have the `JWT_SECRET_KEY` or not

Comment (sharafdin) [reply packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env]: alright. thanks bro

Comment (sharafdin) [reply packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/.env]: > it is no-auth template so it doesn't matter if we have the `JWT_SECRET_KEY` or not

It's important to standardize all of the .env files across the project. We should remove any unnecessary variables from the no-auth .env files to ensure consistency and eliminate clutter. thanks again @Argus416

**Review by sharafdin — APPROVED** [review]
thanks bro @Argus416

**Pull request merged**